app/frontend/src/store/modules/form.js

```diff
@@ -350,9 +350,9 @@ export default {
     },
 
 
-    async deleteMultiSubmissions ({ dispatch },{formId, submissionIds}) {
+    async deleteMultiSubmissions ({ dispatch }, {submissionIds}) {
```

Comment:
I think it's still ok to pass `formId` to check if user have access to form in first place, we should investigate why `canDeleteAndRestoreMultiSubmissions` https://github.com/bcgov/common-hosted-form-service/pull/620/files#diff-c94e18e8dacd854755475f141756a44eb0e9567aabf5df871d930c30a4c4d78eL139 doesn't work as needed...

Reply:
I added the canDeleteAndRestoreMultiSubmissions after the bug was raised. I renamed that this morning to filterMultipleSubmission. I will add formId back